ts/Extensions/Data.ts

```diff
@@ -2201,13 +2237,13 @@ class Data {
                 ) {
                     ret = (
                         (match as any).getTime() -
-                        (match as any).getTimezoneOffset() *
+                        timezoneOffset *
                         60000
                     );
 
                 // Timestamp
                 } else if (isNumber(match)) {
-                    ret = match - (new Date(match)).getTimezoneOffset() * 60000;
+                    ret = match - timezoneOffset * 60000;
```

Comment:
We previously substracted the timezone of the actual date of the match. In the new PR, the `timezoneOffset` is statically set on line 692, which means that it will vary depending on DST or not. As a consequence, check out how a non-UTC summer-time date is parsed (it is winter time as of writing this). Compare this to the results with the PR code http://jsfiddle.net/72kon4ch/